app/assets/javascripts/app/views/profile_sidebar_view.js

```diff
@@ -10,7 +10,7 @@ app.views.ProfileSidebar = app.views.Base.extend({
   },
 
   _shouldShowProfileInfo: function() {
-    return (this.model.isSharing() || this.model.get('is_own_profile'));
+    return true;
```

Comment:
Ok, this is a disputable place. I removed this checks, because they duplicate the same checks, that already have been done in app/presenters/person_presenter.rb (see below) on the server side. @jhass wasn't agree with me on IRC about it. So lets discuss, what is the best way to do it - leave it how I did it, or add check this.model.get('is_public') here just as in PersonPresenter.